push.log

```diff
@@ -0,0 +1,22 @@
+husky > pre-push (node v16.15.0)
```

Comment:
Was this file intended to be part of this PR?

Reply:
No - mistakenly added via Gitpod (my first time using and is a very cool dev environment for small random contributions to projects!)

Reply:
Was this file created by gitpod ? If that's the case we should look at adding it to some gitignore file (either gitpod's or ours).

Thanks for the feedback about using gitpod, we weren't sure this was used actually !

Reply:
It wasn’t; it was a file I created manually while trying to figure out the pre-push hook test results. 

I just added it accidentally due to now being familiar with the gitpod UI! :)

Reply:
Got it, thanks!